rpm.mk

```diff
@@ -0,0 +1,171 @@
+RPM_ARCHITECTURE = x86_64
+HEROKUISH_RPM_PACKAGE_NAME = herokuish-$(HEROKUISH_VERSION)-1.$(RPM_ARCHITECTURE).rpm
+PLUGN_RPM_PACKAGE_NAME = plugn-$(PLUGN_VERSION)-1.$(RPM_ARCHITECTURE).rpm
+SSHCOMMAND_RPM_PACKAGE_NAME = sshcommand-$(SSHCOMMAND_VERSION)-1.$(RPM_ARCHITECTURE).rpm
+SIGIL_RPM_PACKAGE_NAME = gliderlabs-sigil-$(SIGIL_VERSION)-1.$(RPM_ARCHITECTURE).rpm
+
+.PHONY: rpm-all
+
+rpm-all: rpm-setup rpm-herokuish rpm-dokku rpm-plugn rpm-sshcommand rpm-sigil
+	mv /tmp/*.rpm .
+	@echo "Done"
+
+rpm-setup:
+	@echo "-> Installing rpm build requirements"
+	@sudo apt-get update -qq > /dev/null
+	@sudo DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true apt-get install -qq -y gcc git build-essential wget ruby-dev ruby1.9.1 rpm > /dev/null 2>&1
+	@command -v fpm > /dev/null || sudo gem install fpm --no-ri --no-rdoc
+	@ssh -o StrictHostKeyChecking=no git@github.com || true
+
+rpm-herokuish:
+	rm -rf /tmp/tmp /tmp/build $(HEROKUISH_RPM_PACKAGE_NAME)
+	mkdir -p /tmp/tmp /tmp/build
+
+	@echo "-> Cloning repository"
+	git clone -q "https://github.com/$(HEROKUISH_REPO_NAME).git" \
+		--branch "v$(HEROKUISH_VERSION)" /tmp/tmp/herokuish > /dev/null
+	rm -rf /tmp/tmp/herokuish/.git /tmp/tmp/herokuish/.gitignore
+
+	@echo "-> Copying files into place"
+	mkdir -p "/tmp/build/var/lib"
+	cp -rf /tmp/tmp/herokuish /tmp/build/var/lib/herokuish
+
+	@echo "-> Creating $(HEROKUISH_RPM_PACKAGE_NAME)"
+	sudo fpm -t rpm -s dir -C /tmp/build -n herokuish \
+		-v $(HEROKUISH_VERSION) \
+		-a $(RPM_ARCHITECTURE) \
+		-p $(HEROKUISH_RPM_PACKAGE_NAME) \
+		--depends 'docker-engine >= 1.9.1' \
+		--depends 'sudo' \
```

Comment:
This is a debianism, but worth keeping in case CentOs screws the pooch or there is a user out there who thinks they are slick.